tools/Google.Cloud.Tools.VersionCompat.Tests/ClassTest.cs

```diff
@@ -0,0 +1,136 @@
+    namespace FromAbstractNoCtor.B { public class C { private C() { } } }
+
+    namespace FromSealed.A { public sealed class C { public C() { } } }
+    namespace FromSealed.B { public class C { public C() { } } }
+
+    namespace FromSealedNoCtor.A { public sealed class C { private C() { } } }
+    namespace FromSealedNoCtor.B { public class C { private C() { } } }
+
+    namespace BaseTypeFromObjectOk.A { public class C { } }
+    namespace BaseTypeFromObjectOk.B { public class C : List<int> { } }
+
+    namespace BaseTypeFromObjectAbstract.A { public abstract class C { } }
+    namespace BaseTypeFromObjectAbstract.B { public abstract class C : List<int> { } }
+
+    namespace BaseTypeChanged.A { public class C : List<string> { } }
+    namespace BaseTypeChanged.B { public class C : List<int> { } }
+
+    public interface I1 { }
+    internal interface I2 { }
+
+    namespace InterfaceAdded.A { public abstract class C { } }
+    namespace InterfaceAdded.B { public abstract class C : I1 { } }
+
+    namespace InterfaceRemoved.A { public abstract class C : I1 { } }
+    namespace InterfaceRemoved.B { public abstract class C { } }
+
+    namespace NonExportedInterfaceAdded.A { public abstract class C { } }
+    namespace NonExportedInterfaceAdded.B { public abstract class C : I2 { } }
+
+    namespace NonExportedInterfaceRemoved.A { public abstract class C : I2 { } }
+    namespace NonExportedInterfaceRemoved.B { public abstract class C { } }
+
+    namespace GenericConstraintAdded1.A { public class C<A> { } }
+    namespace GenericConstraintAdded1.B { public class C<A> where A : new() { } }
+
+    namespace GenericConstraintAdded2.A { public class C<A> { } }
+    namespace GenericConstraintAdded2.B { public class C<A> where A : class { } }
+
+    namespace GenericConstraintAdded3.A { public class C<A> { } }
+    namespace GenericConstraintAdded3.B { public class C<A> where A : struct { } }
+
+    namespace GenericConstraintAdded4.A { public class C<A> { } }
+    namespace GenericConstraintAdded4.B { public class C<A> where A : IList<A> { } }
+
+    namespace GenericConstraintRemoved1.A { public class C<A> where A : new() { } }
+    namespace GenericConstraintRemoved1.B { public class C<A> { } }
+
+    namespace GenericConstraintRemoved2.A { public class C<A> where A : class { } }
+    namespace GenericConstraintRemoved2.B { public class C<A> { } }
+
+    namespace GenericConstraintRemoved3.A { public class C<A> where A : struct { } }
+    namespace GenericConstraintRemoved3.B { public class C<A> { } }
+
+    namespace GenericConstraintRemoved4.A { public class C<A> where A : IList<A> { } }
+    namespace GenericConstraintRemoved4.B { public class C<A> { } }
+}
+
+namespace Google.Cloud.Tools.VersionCompat.Tests
+{
+    public class ClassTest : TestBase
+    {
+        [Fact] public void NoChange() => TestNone();
+        // TOOD: Enable this test after ctors are dealt with properly.
```

Comment:
Nit: TOOD => TODO, or better, use `[Fact(Skip = "...")]` instead

Reply:
Change to `Skip`